### ISSUE TITLE
WebpackTestRunner.js is malformed

### ISSUE THREAD

**Issue by mark-bradshaw** [thereactivestack/meteor-webpack#164]
TL;DR - `packages/webpack/plugin/WebpackCompiler.js`, lines 763 and 764 are creating a malformed js file when `directories` has more than one item in the array.

We are trying webpack:webpack and immediately ran into issues with testing.  What I determined is that the template string that starts on line 756 has a couple of arrays in the middle of it.  When you concatenate an array with a string you get commas added in the middle.  Normally not a big deal.  But when trying to generate valid javascript, big deal. This only occurs when the array has more than one value in it.

Our WebpackTestRunner.js ends up like this:
```
// This file is auto-generated
// Any change will be overriden
const ignoreTarget = Meteor.isServer ? 'client' : 'server';

let testFiles = [];

if (Meteor.isAppTest) {
  testFiles = testFiles.concat(require.context('./client', true, /.(test|spec|app-test|app-spec)(s)?.(.+)$/i).keys()).map(file => './client' + file.substr(1));
,  testFiles = testFiles.concat(require.context('./coverage', true, /.(test|spec|app-test|app-spec)(s)?.(.+)$/i).keys()).map(file => './coverage' + file.substr(1));
,  testFiles = testFiles.concat(require.context('./imports', true, /.(test|spec|app-test|app-spec)(s)?.(.+)$/i).keys()).map(file => './imports' + file.substr(1));
,  testFiles = testFiles.concat(require.context('./public', true, /.(test|spec|app-test|app-spec)(s)?.(.+)$/i).keys()).map(file => './public' + file.substr(1));
,  testFiles = testFiles.concat(require.context('./server', true, /.(test|spec|app-test|app-spec)(s)?.(.+)$/i).keys()).map(file => './server' + file.substr(1));
} else {
  ....
```

Notice all those lines beginning with commas.  That's the `directories` array on lines 763 and 764 getting concatenated.

I suggest this.  Change lines 763 and 764 to this:
```
${directories.map(directory => `  testFiles = ...stuff...;\n`).join('')}} else {
${directories.map(directory => `  testFiles = ...stuff...\n`).join('')}}
```

Add `.join('')` at the end of the map.  You end up with straight string concatenation, which should take care of the issue.  If you'd like a PR let me know.

**Comment by eXon** [comment]
Thanks for noticing. This is fixed!
